tests/functional/disaster-recovery/regional-dr/test_dryrun_failover.py

```
@@ -0,0 +1,1034 @@
+                info["last_kubeobj_time"] = (
+                    dr_helpers.verify_last_kubeobject_protection_time(
+                        info["drpc_obj"],
+                        info["workload"].kubeobject_capture_interval_int,
+                    )
+                )
+                logger.info(
+                    f"[{info['drpc_resource_name']}] lastKubeObjectProtectionTime = "
+                    f"{info['last_kubeobj_time']} ✓"
+                )
+
+    def _precondition_workloads(
```

Ruff (0.15.20)

[warning] Too many branches (16 > 12)

(PLR0912)